Cmb report auto

bms63 opened the pull request.

bms63 committed:
Create create-cmb-report.yaml

bms63 committed:
Create create-cmb-report.py

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

robertdevine commented:
@bms63 @parmsam-pfizer @RConsortium/pilot5 - attached is today's p5 combined report generated using R only.
[cmb-report-tlf-pilot5.pdf](https://github.com/user-attachments/files/30108510/cmb-report-tlf-pilot5.pdf)

bms63 committed:
Update create-cmb-report.py

bms63 commented:
@robertdevine  you can just upload the R code here as an attachment and I will convert into a github action.  IF not, I guess I will just figure out how to do it in python cause I really want to get this one completed ASAP.  TYTY

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.py

bms63 commented:
> @bms63 @parmsam-pfizer @RConsortium/pilot5 - seems appropriate to use R for consistency in the pilot serries.
>

but we used libreoffice to do this previously.  I don't think it matters how we construct the combined report - I'm happy to use whatever - please just provide.  
 
> [1]R-infra team provides an action yaml template to render R markdown [here](https://github.com/r-lib/actions/tree/v2/examples#render-rmarkdown). [2]In order to capture provenance on the combined report commits by p5 I'll commit the R code here yet linked though PR #88 in that way the full history of commits is transparent.

please just provide here - I am happy to switch the python to R code.  I don't really care what we use.  The branch you are referencing has outdated code in it and I am going to delete it.  The conversations are all recorded if you are worried about transparency.

robertdevine commented:
Attached R code used to generate the p5 combine report [here.](https://github.com/RConsortium/submissions-pilot5-datasetjson/pull/109#issuecomment-3236964346) PR #88 has the initial commit history. 

[cmb-report-tlf-pilot5-R-codes.txt](https://github.com/user-attachments/files/22047741/cmb-report-tlf-pilot5-R-codes.txt)

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Merge branch 'main' into cmb-report-auto

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Remove distutils install, add packaging to pip install in workflow.

bms63 committed:
Delete .github/workflows/create-cmb-report.yaml

bms63 committed:
Create create-cmb-report.yaml

bms63 committed:
removing a bunch of cmb files

Merge remote-tracking branch 'origin/main' into cmb-report-auto

# Conflicts:
#	.github/workflows/create-cmb-report.yaml

bms63 committed:
Merge branch 'cmb-report-auto' of https://github.com/RConsortium/submissions-pilot5-datasetjson into cmb-report-auto

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Update create-cmb-report.yaml

bms63 committed:
Merge branch 'main' into cmb-report-auto

bms63 committed:
Add script to convert RTF to PDF and commit

bms63 committed:
Create convert_rtf_to_pdf.yaml

bms63 committed:
Update convert_rtf_to_pdf.yaml

parmsam-pfizer committed:
Change Python command to use /usr/bin/python3

parmsam-pfizer committed:
Update Python interpreter path in workflow

parmsam-pfizer committed:
Update create-cmb-report.yaml

parmsam-pfizer committed:
Optimize workflow by removing redundant commands

parmsam-pfizer committed:
Remove pip upgrade from Python dependencies installation

parmsam-pfizer committed:
Update create-cmb-report.yaml

parmsam-pfizer committed:
Update create-cmb-report.yaml to remove distutils install

parmsam-pfizer committed:
Update create-cmb-report.yaml to remove pip upgrade

Removed pip upgrade from workflow as it's no longer needed.
